packages/teams-js/test/public/meeting.spec.ts

```diff
@@ -721,11 +721,18 @@ describe('meeting', () => {
             let returnedSdkError: SdkError | null;
             let returnedResult: boolean | null;
```

Comment:
It would be good to add a new e2e test for this as well (or update an existing one). The test definitions for this file are in `meeting.json`. Note: the host sdk change will need to be checked in before the e2e test will work properly

Reply:
Sounds good! Added a new test that passes `shareOptions`. I tested it e2e with my local host sdk changes and the test is passing.